incubator/cloudbeaver/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: cloudbeaver
+description: A web based SQL GUI tool for running queries against NuoDB.
+type: application
+version: 3.6.0
+keywords:
+- nuodb
+- database
+- sql
+home: https://github.com/nuodb/nuodb-helm-charts/tree/master/incubator/cloudbeaver
+maintainers:
+- name: NuoDB
+  email: support@nuodb.com
+engine: gotpl
+icon: https://raw.githubusercontent.com/nuodb/nuodb-helm-charts/master/images/nuodb.svg
+appVersion: 4.3.2
```

Comment:
We might want to change this to the CloudBeaver version that we tested with. I.e `22.1.4`.

Reply:
Good spot, copy-n-paste :-). Fixed.